package.json

```diff
@@ -24,6 +24,8 @@
   },
   "dependencies": {
     "expo": "^20.0.0",
+    "firebase": "^4.6.1",
+    "immutable": "^3.8.2",
```

Comment:
This is not being used now but we will use in future. Right?

Reply:
Yes, I would be pretty surprised if we built an entire RN app without using Immutable at some point.